src/blocks/gallery-carousel/edit.js

```diff
@@ -327,6 +365,32 @@ class GalleryCarouselEdit extends Component {
 						</div>
 					</div>
 				</ResizableBox>
+				<div className={ className }>
```

Comment:
We shouldn't be re-using the main block className here, this className needs to represent the element.

Reply:
> We shouldn't be re-using the main block className here, this className needs to represent the element.

This is used to not have to duplicate much of the carousel styling. It's only within the editor.